plugin/protosizer/size.go

```diff
@@ -0,0 +1,590 @@
+// Copyright (c) 2013, Vastech SA (PTY) LTD. All rights reserved.
```

Comment:
You could probably reuse the code in plugin/size/size.go to generate both Size and ProtoSize methods, just like Equal and VerboseEqual methods are both generated by the equal plugin and marshaler and unsafemarshaler and both generated by the marshaler plugin.